daemon/inertiad/crypto/totp.go

```diff
@@ -0,0 +1,60 @@
+package crypto
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"github.com/pquerna/otp"
+	"github.com/pquerna/otp/totp"
+)
+
+const (
+	TotpIssuerName       = "Inertia"
+	TotpPeriod           = 30
+	TotpSecretSize       = 10
+	TotpDigits           = 6
+	TotpAlgorithm        = otp.AlgorithmSHA1
```

Comment:
Can we use a more secure hash function than SHA1 for this? As I understand it, SHA1 is considered broken for security purposes.

Reply:
needs to be SHA-1 to be compatible with Google Authenticator and probably other TOTP apps

https://medium.com/@tilaklodha/google-authenticator-and-how-it-works-2933a4ece8c2